src/styles/types.js

```diff
@@ -605,6 +605,7 @@ export type IconT = {
   ArrowLeft?: ComponentType<IconPropsT>,
   ArrowRight?: ComponentType<IconPropsT>,
   ArrowUp?: ComponentType<IconPropsT>,
+  Blank?: ComponentType<IconPropsT>,
```

Comment:
why is the blank icon required? does not having it cause the ui to jump?

Reply:
There is a concept of "unsorted" for a column. Base Web doesn't visualize this, but it may be helpful for projects that override the theme to provide an icon for this state. I'm using an empty icon as a placeholder. This prevents the UI from jumping when the sort state changes and serves as the base for the `SortNoneIcon` override.